### PR TITLE
Add Support for http/2 and brotli in webserver tutorial

### PR DESCRIPTION
**Pull request by 2020Sanoj**

### CODE REVIEW

**Comment by 2020Sanoj** [comment]
This PR adds to the tutorial how to enable http/2 in apache2

**Review by raphi-0901 — CHANGES_REQUESTED** [review]

**Comment by raphi-0901** [comment]
Hey, where is the screenshot for Step 1.6? There you say "see screenshot below", but there is no screenshot.

Also, in order to finish the Nextcloud installation, I had to change the nextcloud root folder´s permissions. In addition to that, when I try to visit the Nextcloud page on my server, it just keeps loading over and over.

File: community-tutorials/install-web-server-mariadb-debian-php-wordpress-yourls-nextcloud-uptimekuma/01-en.md [hunk 1]
```diff
@@ -196,25 +215,39 @@ In the following commands, replace every `DOMAIN` with your domain or subdomain:
...
 </VirtualHost>
 ```
 
 - Replace every `DOMAIN` with your domain and set a folder path under "DocumentRoot". All files in this folder will be available on your DOMAIN in the web.<br/>
 - Save your changes by pressing <kbd>CTRL + X</kbd>, then <kbd>y</kbd> and finally by hitting <kbd>ENTER</kbd>. <br/>
 - Now create this folder with `mkdir /var/www/FOLDER`. <br>
-- Restart Apache2 with `service apache2 restart.` <br>
+- Restart Apache2 with `service apache2 restart`. <br>
 
-If you are doing this directly on a domain and NOT on a subdomain, I recommend repeating this step with www.DOMAIN (you can paste it in the same file, but leave some lines blank).
```
[the remark this answers is not in the document]
Comment (2020Sanoj) [reply community-tutorials/install-web-server-mariadb-debian-php-wordpress-yourls-nextcloud-uptimekuma/01-en.md:212]: i replaced this line

**Comment by 2020Sanoj** [comment]
> Hey, where is the screenshot for Step 1.6? There you say "see screenshot below", but there is no screenshot.

There stands nothing about a screenshot, I mean the text below

**Comment by 2020Sanoj** [comment]
> Hey, where is the screenshot for Step 1.6? There you say "see screenshot below", but there is no screenshot.
> 
> Also, in order to finish the Nextcloud installation, I had to change the nextcloud root folder´s permissions. In addition to that, when I try to visit the Nextcloud page on my server, it just keeps loading over and over.

The permissions are now automatically for every file inside the folder correct and for the second did you reinstalled nextcloud or did you used an old installation?

**Comment by raphi-0901** [comment]
> > Hey, where is the screenshot for Step 1.6? There you say "see screenshot below", but there is no screenshot.
> 
> There stands nothing about a screenshot, I mean the text below

Oh  sorry, I mean Step 1.7

**Comment by raphi-0901** [comment]
> > Hey, where is the screenshot for Step 1.6? There you say "see screenshot below", but there is no screenshot.
> > Also, in order to finish the Nextcloud installation, I had to change the nextcloud root folder´s permissions. In addition to that, when I try to visit the Nextcloud page on my server, it just keeps loading over and over.
> 
> The permissions are now automatically for every file inside the folder correct and for the second did you reinstalled nextcloud or did you used an old installation?

I didn´t use an old installation.

**Comment by raphi-0901** [comment]
Ok, I reinstalled Nextcloud again and it works fine now!

**Comment by raphi-0901** [comment]
Hey @2020Sanoj, after discussion in our team we came to the decision that the tutorial is too big in size and it combines too many different topics. We would be happy if you could resubmit the tutorial in smaller parts. For example, one for Wordpress, one for Nextcloud, ... .

**Comment by 2020Sanoj** [comment]
So one for the complete web and database setup and four more for WordPress, Nextcloud, YOURLS and uptime Kama?

**Pull request closed without merging**